sdk/src/main/java/com/karhoo/sdk/api/service/loyalty/LoyaltyBurnPointsInteractor.kt

```diff
@@ -0,0 +1,33 @@
+package com.karhoo.sdk.api.service.loyalty
+
+import com.karhoo.sdk.api.KarhooError
+import com.karhoo.sdk.api.datastore.credentials.CredentialsManager
+import com.karhoo.sdk.api.model.LoyaltyPoints
+import com.karhoo.sdk.api.network.client.APITemplate
+import com.karhoo.sdk.api.network.response.Resource
+import com.karhoo.sdk.api.service.common.BaseCallInteractor
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.Dispatchers
+import javax.inject.Inject
+import kotlin.coroutines.CoroutineContext
+
+internal class LoyaltyBurnPointsInteractor @Inject constructor(credentialsManager: CredentialsManager,
+                                                           private val apiTemplate: APITemplate,
+                                                           context: CoroutineContext = Dispatchers.Main
+                                                          ) :
+    BaseCallInteractor<LoyaltyPoints>(true, credentialsManager, apiTemplate, context) {
```

Comment:
Boolean literal argument is without parameter name. Maybe add the param name also:
requestRequiresToken = true